index.js

```diff
@@ -66,7 +66,55 @@ function toggleColor(element) {
         element.style.color = "green"
     }
 }
+const changeColor = document.querySelector("#header")
+changeColor.addEventListener('click', function (event){
+    // console.log(toggleColor)
+    if (changeColor.style.color === "green"){
+        changeColor.style.color = "black"
+    } else {
+        changeColor.style.color = "green"
+    }
+})
+     
 
 /***** Deliverable 2 *****/
+// const likeButton = document.querySelector("#profile > button")
+
+// likeButton.addEventListener('click', function(likes){
+
+
+// traveler.likes += 1 
+
+// likeCounter.innerHTML = `${traveler.likes} Likes`
+// })
+
+const likesButton = document.querySelector("#profile > button")
+function addLike (){
+    likeCounter = document.querySelector("#profile > p")
+    traveler.likes+=1
```

Comment:
Remember we don't want this information to persist - meaning we should only see it update on the DOM and if we refresh, the likes get reset. I think you're on the right track by finding the likeCounter p tag. What could we do to ONLY update that number?

You were on the right track by defining our likes <p> tag on line 75! Because remember - we don't want this information to persist and update in our traveler object.

If i separated this out into its own function it might look something like this:

const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling; /// this can account for if there are multiple travelers with likes, but in this instance your `likeCounter` variable would work as well.
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)

Reply:
It's a bit more work, but we definitely want to be mindful of how we can directly manipulate the DOM as opposed to our data that's being reflected on the DOM